### DISCUSSION TITLE
Delete Series API Endpoint in Mimir

### DISCUSSION THREAD

**Discussion by miladjahandideh** [grafana/mimir#3835] [Help and support]
Is there an API endpoint for `Delete Series` like [Prometheus Delete Series](https://prometheus.io/docs/prometheus/latest/querying/api/#delete-series)? I want to delete the all-time history of one metric.

**Comment by pracucci** [comment]
👋 Series deletion is currently not supported by Mimir (there's an issue here https://github.com/grafana/mimir/issues/88). Mimir still supports per-tenant retention, but not individual series deletion.

**Comment by gfonseca-tc** [comment]
A bit desperating to hear that after you have 12 months of production data in mimir and needs to fix some wrongly uploaded data... Would love to see this feature implemented.
